README.md

```diff
@@ -78,3 +80,8 @@ Then the following format is available to set the UID/GID:
 ```sh
 -u, --user=                Username or UID (format: <name|uid>[:<group|gid>])
 ```
+
+[travis-badge]: https://travis-ci.org/xenserver/xenserver-build-env.png?branch=master
+[travis-url]: https://travis-ci.org/xenserver/xenserver-build-env
+[docker-pulls-badge]: https://img.shields.io/docker/pulls/jekyll/jekyll.svg
```

Comment:
Oops that was a test. It was late. Will fix.

-- simon.beaumont@citrix.com (via iPhone)

On 15 Jun 2016, at 01:05, John Else <notifications@github.com<mailto:notifications@github.com>> wrote:


In README.md<https://github.com/xenserver/xenserver-build-env/pull/67#discussion_r67079766>:

> @@ -78,3 +80,8 @@ Then the following format is available to set the UID/GID:
>  ```sh
>  -u, --user=                Username or UID (format: <name|uid>[:<group|gid>])
>  ```
> +
> +[travis-badge]: https://travis-ci.org/xenserver/xenserver-build-env.png?branch=master
> +[travis-url]: https://travis-ci.org/xenserver/xenserver-build-env
> +[docker-pulls-badge]: https://img.shields.io/docker/pulls/jekyll/jekyll.svg


jekyll? ?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/xenserver/xenserver-build-env/pull/67/files/05dc0042ef8f36f0a39f0821d520cf4a582ff727#r67079766>, or mute the thread<https://github.com/notifications/unsubscribe/ABEso7oumPFLxYky7Q4CogbpHT6EdQpjks5qL0HKgaJpZM4I11TY>.